src/components/VideoPopoverMenu/index.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import PopoverMenu from '@components/PopoverMenu';
+import {useVideoPopoverMenuContext} from '@components/VideoPlayerContexts/VideoPopoverMenuContext';
+
+const propTypes = {
+    isPopoverVisible: PropTypes.bool,
+
+    hidePopover: PropTypes.func,
+
+    anchorPosition: PropTypes.shape({
+        horizontal: PropTypes.number.isRequired,
+        vertical: PropTypes.number.isRequired,
+    }),
+};
+const defaultProps = {
+    isPopoverVisible: false,
+    anchorPosition: {
+        horizontal: 0,
+        vertical: 0,
+    },
+    hidePopover: () => {},
+};
+
+function VideoPopoverMenu({isPopoverVisible, hidePopover, anchorPosition}) {
+    const {menuItems} = useVideoPopoverMenuContext();
+
+    return (
+        <PopoverMenu
+            onClose={hidePopover}
+            onItemSelected={hidePopover}
+            isVisible={isPopoverVisible}
+            anchorPosition={anchorPosition}
+            menuItems={menuItems}
+            withoutOverlay
```

Comment:
There is a much better feeling when this popover is without overlay because a user can interact with all other buttons even if the popover is opened. However, if it isn't a problem for Expensify and we can turn this off, let's do this then :D